.gitignore

```diff
@@ -6,7 +6,6 @@
 /public/bundles/
 /var/
 /vendor/
-/public/cover
```

Comment:
This should be re-add to ensure upload covers from local is not commit into github